index.ts

```diff
@@ -315,6 +315,233 @@ export async function createElasticsearchMcpServer(
     }
   );
 
+  // Tool 4: Get shard information
+  server.tool(
+    "get_shards",
```

Comment:
Please only include changes relevant to PR, the get_shards tool is added as part of #25

Reply:
I apologize for not creating a separate branch from the beginning, which led to this dependency. I will be more careful with branch management in future contributions.